src/Controls/src/Core.Design/FlowDirectionDesignTypeConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Maui.Controls.Design
+{
+	public class FlowDirectionDesignTypeConverter : KnownValuesDesignTypeConverter
+	{
+		protected override bool ExclusiveToKnownValues => true;
+
+		protected override string[] KnownValues
+			=> new[]
+			{
+				"MatchParent",
+				"LeftToRight",	
+				"RightToLeft",
+				"inherit",
+				"ltr",
+				"rtl"
+			};
```

Comment:
@Redth You just need to remove inherit, ltr, and rtl from this DT converter and the XAML editor complain about anything that isn't one of the first three values.